src/main/java/uk/gov/hmcts/divorce/citizen/notification/SaveAndSignOutNotificationHandler.java

```diff
@@ -36,6 +37,7 @@
     public void notifyApplicant(CaseData caseData) {
         Map<String, String> templateVars = new HashMap<>();
         templateVars.put(FIRST_NAME, caseData.getPetitionerFirstName());
+        templateVars.put(MIDDLE_NAME, caseData.getPetitionerMiddleName());
```

Comment:
do you display middle name in the notify email ? And is the gov notify email template updated ?

Reply:
Oh right, I don't think so. At least not yet